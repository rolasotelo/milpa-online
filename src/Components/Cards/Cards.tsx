import React from "react";
import useGameContext from "../../Hooks/useGameContext/useGameContext";
import CropCard from "../CropCard/CropCard";

interface Props {}

const exampleCards = [
  { title: "Corn", description: "Good for esquites" },
  { title: "Tomate", description: "De nada Italia" },
  { title: "Chile", description: "Calieeentee" },
];

const Cards = (props: Props) => {
  const context = useGameContext();
  return (
    <div className="flex flex-col bg-mexicanBlue w-1/3 p-10 justify-evenly">
      {context.cropsHand.map((crop, index) => {
        return <CropCard key={index} title={crop?.name} />;
      })}
    </div>
  );
};

export default Cards;
