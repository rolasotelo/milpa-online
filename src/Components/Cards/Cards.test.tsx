import React from "react";
import renderer from "react-test-renderer";

import Cards from "./Cards";

describe("<Cards />", () => {
  const defaultProps = {};
  const wrapper = renderer.create(<Cards {...defaultProps} />);

  test("render", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
