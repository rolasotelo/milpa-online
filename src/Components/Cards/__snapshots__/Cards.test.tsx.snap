// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Cards /> render 1`] = `
<div
  className="flex flex-col bg-mexicanBlue w-1/3 p-10 justify-evenly"
>
  <div
    className="bg-mexicanBone  h-28"
  >
    Corn
  </div>
  <div
    className="bg-mexicanBone  h-28"
  >
    Tomate
  </div>
  <div
    className="bg-mexicanBone  h-28"
  >
    Chile
  </div>
</div>
`;
